README.md

```diff
@@ -87,14 +87,25 @@ api = Bigcommerce::Api.new({
   :store_url => "https://store.mybigcommerce.com",
   :username  => "username",
   :api_key   => "api_key",
-  :ssl_client_cert  =>  OpenSSL::X509::Certificate.new(File.read("cert.pem")),
-  :ssl_client_key   =>  OpenSSL::PKey::RSA.new(File.read("key.pem"), "passphrase, if any"),
-  :ssl_ca_file      =>  "ca_certificate.pem",
-  :verify_ssl       =>  OpenSSL::SSL::VERIFY_PEER
+  :ssl_client_cert  =>  "/path/to/cert.pem",
+  :ssl_client_key   =>  { path: "/path/to/key.pem", passphrase: "passphrase, if any" },
+  :ssl_ca_file      =>  "/path/to/ca_certificate.pem",
+  :verify_peer      =>  OpenSSL::SSL::VERIFY_PEER
+})
+```
+
+Disabling the SSL verification is only advisable in development enironment and it can be done by explicitly passing `:verify_peer` field as below:
+
+```rb
+require 'bigcommerce'
+
+api = Bigcommerce::Api.new({
```

Comment:
@pedelman That's certainly a good suggestion from production environment point of view where API user is *explicitly* specifying SSL options. But to connect to BigCommerce through Legacy API we have 2 ways one *without* supplying SSL options and other *with* SSL options (even though in both cases SSL is verified).  To disable verification we just need `verify_peer` and other options are not mandatory. 
Let me know what you would prefer to go with and we could work on the same.